mantidproject/mantid
# Indirect Bayes - Plot and save via interface

The BayesQuasi algorithm previously contained methods to save and plot the result workspace. This has been refactored to the Quasi tab in the Indirect Bayes interface.
Also written function plotSpectra in IndirectTab which takes a list of workspaces and a given set of indices and plots the respective indices from each workspace.

**To test:**

Indirect > Bayes > Quasi
File: irs26176_graphite002_red.nxs
Resolution: irs26173_graphite002_res.nxs
Click "save" button
Check save directory for:
irs26176_graphite002_QLr_Workspaces.nxs
irs26176_graphite002_QLr_Result.nxs
Also click "plot" button with a selected option

Fixes #17012 .

[RELEASE NOTES](https://github.com/mantidproject/mantid/blob/4c96e32c7ead91e78291b4462ca3f985d2116373/docs/source/release/v3.8.0/indirect_inelastic.rst)

---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.

Refs #17012
